rest/taskrouter/taskqueues/list/post/example-1/example-1.5.x.cs

```diff
@@ -0,0 +1,24 @@
+// Download the twilio-csharp library from
+// https://www.twilio.com/docs/libraries/csharp#installation
+using System;
+using Twilio;
+using Twilio.Rest.Taskrouter.V1.Workspace;
+
+class Example
+{
+    static void Main(string[] args)
+    {
+        // Find your Account Sid and Auth Token at twilio.com/console
+        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string authToken = "your_auth_token";
+        const string workspaceSid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+
+        TwilioClient.Init(accountSid, authToken);
+
+        var taskQueue = TaskQueueResource.Create(
+            workspaceSid, "English", "WAxxxx", "WAyyyy",
```

Comment:
Using extra variables that explain what are "WAxxxx" and "WAyyyy" would be really helpful. What do you think?